### ISSUE TITLE
CGBitmapContextCreate: unsupported parameter combination

### ISSUE THREAD

**Issue by sprzenus** [SDWebImage/SDWebImage#3333]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | ios
 Platform Version        | 15.2
 SDWebImage Version      | 5.12.4
 Integration Method      | cocoapods
 Xcode Version           | Xcode 13.2.1
 Repro rate              | all the time (100%)
 Repro with our demo prj | IDK, it doesn't work on my mac
 Demo project link       | I don't have time for that right now

### Issue Description and Steps
Using the function:
```swift
imageView.sd_setImage(
    with: optionalUrl,
    placeholderImage: placeholderImage,
    options: [],
    completed: nil
)
```
When requesting a PNG image from the URL, receiving the warning/error in the console:

```
[Unknown process name] 
CGBitmapContextCreate: unsupported parameter combination:
 	8 bits/component; integer;
 	32 bits/pixel;
	RGB color space model; kCGImageAlphaLast;
	default byte order;
	416 bytes/row.
Valid parameters for RGB color space model are:
	16  bits per pixel,		 5  bits per component,		 kCGImageAlphaNoneSkipFirst
	32  bits per pixel,		 8  bits per component,		 kCGImageAlphaNoneSkipFirst
	32  bits per pixel,		 8  bits per component,		 kCGImageAlphaNoneSkipLast
	32  bits per pixel,		 8  bits per component,		 kCGImageAlphaPremultipliedFirst
	32  bits per pixel,		 8  bits per component,		 kCGImageAlphaPremultipliedLast
	32  bits per pixel,		 10 bits per component,		 kCGImageAlphaNone|kCGImagePixelFormatRGBCIF10
	64  bits per pixel,		 16 bits per component,		 kCGImageAlphaPremultipliedLast
	64  bits per pixel,		 16 bits per component,		 kCGImageAlphaNoneSkipLast
	64  bits per pixel,		 16 bits per component,		 kCGImageAlphaPremu2022-03-14 15:33:13.076666+0100 Mentorist[9338:2888790] [Unknown process name] clip: empty path.
ltipliedLast|kCGBitmapFloatComponents|kCGImageByteOrder16Little
	64  bits per pixel,		 16 bits per component,		 kCGImageAlphaNoneSkipLast|kCGBitmapFloatComponents|kCGImageByteOrder16Little
	128 bits per pixel,		 32 bits per component,		 kCGImageAlphaPremultipliedLast|kCGBitmapFloatComponents
	128 bits per pixel,		 32 bits per component,		 kCGImageAlphaNoneSkipLast|kCGBitmapFloatComponents
 See Quartz 2D Programming Guide (available online) for more information.
 ```
This didn't happen in version 5.12.3.

**Comment by randarnold** [comment]
I see this as well, although I gotten around to pinning down the source of the log.

**Comment by arshiacont** [comment]
Same here.. We see this since 5.12.4

**Comment by dreampiggy** [comment]
It's OK to ignore this warning.
See #3332

**Comment by dreampiggy** [comment]
If this is annoying, I can provide a better hack for Apple.

If target CGBitmapFormat == BGRA8888, then hardcode to use ARGB8888.

**Comment by dreampiggy** [comment]
Should be fixed by [5.12.5](https://github.com/SDWebImage/SDWebImage/releases/tag/5.12.5)
